cloud/pkg/taskmanager/nodeupgradecontroller/node_upgrade_controller.go

```diff
@@ -210,7 +210,14 @@ func patchStatus(nodeUpgrade *v1alpha1.NodeUpgradeJob, status v1alpha1.NodeUpgra
 	klog.V(4).Info("patch upgrade task status result: ", result)
 	return nil
 }
-
+func (ndc *NodeUpgradeController) GetTaskNodes(name string) ([]string, error) {
+	nodeUpgradeJob, err := ndc.CrdClient.OperationsV1alpha1().NodeUpgradeJobs().Get(context.TODO(), name, metav1.GetOptions{})
+	if err != nil {
+		klog.Infof("NodeUpgradeTask %s not exist", name)
```

Comment:
The error not exist here does not cover all error scenarios. BTW, you can use wrap error to handle the error message instead of logger message.
```golang
return nil, fmt.Errorf("xxxxx, err: %v", err)
```